engines/group/app/controllers/group/api/v1/groups_controller.rb

```diff
@@ -55,10 +68,34 @@ def group_params
         end
 
         def find_and_authorize_group
-          @group = ::Group::Group.find(params[:id])
+          @group = ::Group::Group.find_by_id(params[:id])
 
           authorize @group
         end
+
+        def initialize_side_effects
+          @side_effects = []
+        end
+
+        # TODO: Extract this to lib or gem
+        #
+        def add_side_effects_links
+          return unless @side_effects.any
+
+          links = @side_effects.flatten.map do |side_effect|
+            "<#{api_link(side_effect)}>; rel=\"created\" type=\"#{model_type(side_effect)}\""
+          end.join(', ')
+
+          response.headers['Link'] = links
+        end
+
+        def api_link(side_effect)
+          "#{web_app_url}/api/v1/#{model_type(side_effect).downcase.pluralize}/#{side_effect.id}"
+        end
+
+        def model_type(side_effect)
+          side_effect.class.name
+        end
```

Comment:
Maybe we can wait to add `side_effects || resource_links` in another iteration. I think for onboarding we don't need it. We don't need merbership of the user. 

The next think after create group is bulk invite. When we post to `/invitations/bulk` we check by user_id if user can or not bulk invite. So it's not necessary return membership on group creation. 

Let's talk offline. Maybe I'm missing something